chia/full_node/full_node_rpc_api.py

```diff
@@ -807,16 +810,18 @@ async def get_additions_and_removals(self, request: dict[str, Any]) -> EndpointR
         if "header_hash" not in request:
             raise ValueError("No header_hash in request")
         header_hash = bytes32.from_hexstr(request["header_hash"])
-
-        block: Optional[FullBlock] = await self.service.block_store.get_full_block(header_hash)
-        if block is None:
+        block_bytes: Optional[bytes] = await self.service.block_store.get_full_block_bytes(header_hash)
+        if block_bytes is None:
             raise ValueError(f"Block {header_hash.hex()} not found")
 
+        block_view = memoryview(block_bytes)
+        height, _is_tx_block = get_height_and_tx_status_from_block(block_view)
+
```

Comment:
We should be able to avoid this extra complexity by simply augmenting `get_full_block_bytes` to return the height as well. We have it in the `full_blocks` table it already queries.